src/ui/pages/SidePage/components/IncomingRequest/components/CredentialRequest.tsx

```diff
@@ -21,6 +22,9 @@ const CredentialRequest = ({
   handleCancel,
 }: RequestProps) => {
   const fallbackLogo = KeriLogo;
+  if (requestData.type !== IncomingRequestType.CREDENTIAL_OFFER_RECEIVED) {
```

Comment:
I've got an idea here:
- Make the type a generic on `RequestProps`
- Also remove the last field `incomingRequestType?: string;` - I'm not sure if this is used but if it is, we should just be using `requestData.type`
```
type RequestProps<T extends IncomingRequestType> = {
  pageId: string;
  activeStatus?: boolean;
  blur?: boolean;
  setBlur?: (value: boolean) => void;
  requestData: IncomingRequestProps & { type: T };
  initiateAnimation: boolean;
  handleAccept: () => void;
  handleCancel: () => void;
  handleIgnore?: () => void;
}
```

Now, here you can do `RequestProps<IncomingRequestType.CREDENTIAL_OFFER_RECEIVED>)` and the type system protects us and we can remove this if statement.

Let me know if this works.